### PR TITLE
JNG-4388 Add transfer and service definition to syntax highlight

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4388" title="JNG-4388" target="_blank">TaskJNG-4388</a>  Add to JSL tmBundle the service definitions
  </td></table>
  <br />

### CODE REVIEW

File: Syntaxes/jsl.tmLanguage [hunk 1]
```diff
@@ -1311,89 +2323,285 @@
...
+			</array>
+		</dict>
 	</dict>
 	<key>scopeName</key>
 	<string>source.jsl</string>
 	<key>uuid</key>
 	<string>9E6E7290-F2E8-4D58-97E2-08CBBB3EB7E8</string>
 </dict>
-</plist>
+</plist>
```
Comment (robertcsakany) [inline Syntaxes/jsl.tmLanguage]: :)

**Review by robertcsakany — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]
yes

**Review by noherczeg — CHANGES_REQUESTED** [review]
Please add a few screenshots!

**Comment by gaborflorian** [comment]
Screenshot 2023-04-14 at 16 40 29

**Comment by gaborflorian** [comment]
Screenshot 2023-04-14 at 16 42 24

**Comment by gaborflorian** [comment]
Screenshot 2023-04-14 at 16 43 50

**Review by noherczeg — APPROVED** [review]

**Pull request merged**